lib/addons/src/index.js

```diff
@@ -2,7 +2,7 @@ export class AddonStore {
   constructor() {
     this.loaders = {};
     this.panels = {};
-    this.channel = null;
+    this.channel = { on() {}, emit() {} };
```

Comment:
I have to agree with @shilman here, it's better that our or even a user's tests fail than they can't actually run storybook or an addon in the first place and can't figure out why (it's hard enough with the channel undefined error in the first place).